content/en/Getting started/Assets/_index.md

```diff
@@ -50,3 +50,6 @@ The asset screen prompts you for the following information:
 
 The UI provides the information that you need to add an Asset Image and Title.
 Now take the next step and define your [Asset Type](./asset-type).
+
+<!-- For content, see https://github.com/cobalthq/cobalt-product-public-docs/blob/main/layouts/shortcodes/invite-help.md -->
```

Comment:
Until the PR is merged, you'll see this file in the following directory: https://github.com/cobalthq/cobalt-product-public-docs/tree/DOCS-70_incorporate_5steps_article_from_zendesk/layouts/shortcodes